common/views/components/BasePage/EventPage.js

```diff
@@ -169,6 +169,9 @@ const EventPage = ({ event }: Props) => {
     topLink={{text: 'Events', url: '/events'}}
   />);
 
+  const ticketButtonText = 'Check for tickets';
+  const ticketButtonLoadingText = 'Checking…';
```

Comment:
Not convinced 'Checking…' is the right messaging for this (i.e. waiting for the iframe to load). Maybe, 'Wait…' or 'Loading…' or 'Give us a second…'?

Reply:
ping @Heesoomoon

Reply:
Could we have consistent messaging for this? In that case maybe `Loading…` works.

Reply:
CHanged.